parser/Builder.cc

```diff
@@ -1209,6 +1269,10 @@ class Builder::Impl {
                parser::isa_node<DSymbol>(&node) || parser::isa_node<Regexp>(&node) || parser::isa_node<Array>(&node) ||
                parser::isa_node<Hash>(&node);
     }
+
+    bool isNumberedParameterName(std::string name) {
```

Comment:
Can this take a `string_view` instead? That will avoid allocating and deallocating a `std::string`.